packages/fluentui/docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -112,7 +112,9 @@ class ComponentDoc extends React.Component<ComponentDocProps, ComponentDocState>
       onRemove: item => `${item} has been removed.`,
     };
 
-    const getA11yStatusMessage = ({ isOpen, itemToString, previousResultCount, resultCount, selectedItem }) => {
+    // ToDo: remove default parameter from previousResultCount when issue is fixed
+    // https://github.com/downshift-js/downshift/issues/999
+    const getA11yStatusMessage = ({ isOpen, itemToString, previousResultCount = -1, resultCount, selectedItem }) => {
```

Comment:
This looks like a breaking change

Reply:
unfortunately yes, I messed that up, will create a PR with the fix for the issue and will get the typing back to normal.

Reply:
So, can we provide a temporary fix on our side to avoid breaking changes? I mean to have a wrapper around this function in `Dropdown`

Reply:
Or we can update all the way to 5.2.0 where this is fixed